hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/ozoneimpl/OzoneContainer.java

```diff
@@ -133,6 +138,16 @@ public OzoneContainer(
     volumeSet.setFailedVolumeListener(this::handleVolumeFailures);
     metaVolumeSet = new MutableVolumeSet(datanodeDetails.getUuidString(), conf,
         context, VolumeType.META_VOLUME, volumeChecker);
+    if (VersionedDatanodeFeatures.SchemaV3.chooseSchemaVersion()
+        .equals(OzoneConsts.SCHEMA_V3)) {
+      dbVolumeSet = HddsServerUtil.getDatanodeDbDirs(conf).isEmpty() ? null :
```

Comment:
Shutdown this dbVolumeSet in #stop function.

Reply:
1. Oh, yes, we should shutdown there
2. Sure.
3. That's a good idea, we could have a new scanner to check the db instances in another PR.

Reply:
I created HDDS-6616 to track this new db scanner requirement.